python/utils/worker.py

```diff
@@ -488,7 +514,57 @@ def run(self):
                 self.processing_result = self.processor.process_transactions(
                     self.transactions, start_version, end_version
                 )
-
+                processor_name = self.processor.name()
+                start_version = str(start_version)
+                end_version = str(end_version)
+                num_of_transactions = str(end_version - start_version + 1)
+                processing_duration_in_secs = format(
+                    self.processing_result.processing_duration_in_secs, ".8f"
+                )
+                db_insertion_duration_in_secs = format(
+                    self.processing_result.db_insertion_duration_in_secs, ".8f"
+                )
+                duration_in_secs = format(
+                    self.processing_result.processing_duration_in_secs
+                    + self.processing_result.db_insertion_duration_in_secs,
+                    ".8f",
+                )
+                logging.info(
```

Comment:
Do we want to hide this behind `enable_verbose_logging` config flag?

Reply:
done.

Reply:
ack; we can revisit this after the config type is fixed